### PR TITLE
Add basic packaging

### PR DESCRIPTION
**Pull request by jcdyer**
## Description:

Added some simple packaging integration.
Includes:
- .travis.yml
- requirements.txt
- tox.ini
- pylintrc

This is pulled from https://github.com/edx/cookiecutter-django-app, but stripped down to get us moving quickly.  We will circle back on this and use the full cookiecutter once there is one oriented toward independent libraries.
### Reviewers
- [ ] @nasthagiri 
- [ ] @jmbowman 

FYI @sreecharan93 
### TODO post review:
- [ ] Squash and merge

### CODE REVIEW

**Comment by nasthagiri** [comment]
👍  ok for now.

File: .travis.yml
Comment (jmbowman) [inline .travis.yml]: Do you want to add codecov and deployment support now? https://github.com/edx/cookiecutter-django-app/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/.travis.yml

File: pylint_tweaks
Comment (jmbowman) [inline pylint_tweaks]: Do you actually need this `disable` setting with pylint_django having been removed from `load-plugins` above?  I didn't need it in the edx-sphinx-theme repo, which also doesn't use Django

File: pylint_tweaks
Comment (jmbowman) [inline pylint_tweaks]: Shouldn't this file be named `pylintrc_tweaks`?  Its content isn't reflected in pylintrc below

File: requirements.txt
Comment (jmbowman) [inline requirements.txt:9]: Not using pip-tools yet?  It's pretty handy even if you just want a single requirements.in/.txt pair.

Also, `tox-battery` is a pretty handy package to include (detects changes in the requirements file so that tox virtualenvs get recreated when appropriate)

File: tox.ini
Comment (jcdyer) [inline tox.ini]: These files should be moved into a common notifications package, but I don't want to make that change on this branch, because it will cause merge problems with the existing work.

File: tox.ini
Comment (jmbowman) [inline tox.ini:6]: Might want to add `exclude = .git,.tox` here

File: tox.ini
Comment (jmbowman) [inline tox.ini]: Are all the Python files really going to be in the root directory of the repo?

**Review by jmbowman — CHANGES_REQUESTED** [review]
Looks like a good start, just spotted a few things that you might want to tweak before merging.

File: .travis.yml
Comment (jcdyer) [reply .travis.yml]: Codecov, yes.  Deployment, not yet.

File: pylint_tweaks
Comment (jcdyer) [reply pylint_tweaks]: Thanks.  Fixed.

File: pylint_tweaks
Comment (jcdyer) [reply pylint_tweaks]: Probably a side effect of misnaming the file.

File: requirements.txt
Comment (jcdyer) [reply requirements.txt:9]: The only actual app dependency at the moment is pytz, so we'll hold off on pip-tools

tox-battery will be useful.  Adding.

File: tox.ini
Comment (jcdyer) [reply tox.ini:6]: Good call.

File: tox.ini
Comment (jcdyer) [reply tox.ini]: This needs to be cleaned up, but I want that to happen in #1, before we fix it in the tox.ini, so I don't cause terrible merge conflicts.

**Comment by jcdyer** [comment]
@jmbowman FYI: The goal here is to get test and linting feedback on the repo as quickly as possible for @sreecharan93.  I've made several of the updates you suggested, but held off on others.  We will circle back and make this more robust before this hits production use.

**Review by jmbowman — APPROVED** [review]

**Comment by jmbowman** [comment]
Sure, sounds good.

**Pull request merged**
